Services/DataBaseService/Database/Migrations/Scripts/05.2020/0001_CreateBotsTable.sql

```diff
@@ -0,0 +1,10 @@
+﻿CREATE TABLE [dbo].[Bots](
```

Comment:
We have isRunning but what if person delete bot? Do we can achieve this information?